mozilla_version/gecko.py

```diff
@@ -66,7 +67,7 @@
     |b(?P<beta_number>\d+)
     |(?P<is_esr>esr)
 )?
-(build(?P<build_number>\d+))?$""", re.VERBOSE)
+-?(build(?P<build_number>\d+))?$""", re.VERBOSE)
```

Comment:
I understand this change, and am fine with it.

However, it makes me a little sad that we need to be liberal in what format we expect here.

Reply:
Yeah, it makes me sad too. On the bright side, we're just allowing a dash to be present or not. The data contained in the string isn't changed.